rustls/src/manual/fips.rs

```diff
@@ -1,3 +1,4 @@
+// ORPHANED DUE TO FEATURE NOW REMOVED FROM THIS FORK: fips
```

Comment:
added in response to: https://github.com/brody4hire/portable-rustls/pull/7/files#r1943949622

I think this is a bit ugly but at least better than nothing.

Reply:
```suggestion
// ORPHANED DUE TO FEATURE REMOVED FROM THIS FORK: fips
```

(should slightly improve consistency with other updates coming)